Jenkinsfile

```diff
@@ -469,11 +470,12 @@ EOF"""
           } // end of timeout - post install verify script
 
           // pull job log
-          sh "./scripts/show-job-logs.sh -d -H '${SSH_HOST}' -P '${zOsmfPort}' -u '${USERNAME}' -p '${PASSWORD}' -n 'ZWES*STC' -o ZWESVUSR -a file-contents"
+          sh "./scripts/show-job-logs.sh -d -H '${SSH_HOST}' -P '${zOsmfPort}' -u '${USERNAME}' -p '${PASSWORD}' -n 'ZWESECUR' -o '${USERNAME}' -a file-contents"
+          sh "./scripts/show-job-logs.sh -d -H '${SSH_HOST}' -P '${zOsmfPort}' -u '${USERNAME}' -p '${PASSWORD}' -n 'ZWES*' -o 'ZWES*' -a file-contents"
```

Comment:
I think this needs updating to 'ZWES*STC' and probably a XMEM version run as it looks like is shows the job log of the first job that matches, so only ZWESECUR shows in the console at the moment

Reply:
This is using z/OSMF jobs api in the backend. So the `-n 'ZWES*' -o 'ZWES*'` pattern will match jobs start with `ZWES`, which is `ZWESISTC`. It won't list `ZWE1SV` the Zowe job. So we may need to change to `ZWE*` to show all Zowe jobs. `ZWES*STC` also can only match `ZWESISTC`, but will not show Zowe job `ZWE1SV`.